src/test/run-make-fulldeps/instrument-coverage/Makefile

```diff
@@ -0,0 +1,67 @@
+# needs-profiler-support
+
+-include ../tools.mk
+
+# This test makes sure that LLVM coverage maps are genereated in LLVM IR.
+
+COMMON_FLAGS=-Zinstrument-coverage
+
+# LLVM doesn't yet support instrumenting binaries that use unwinding on MSVC:
+# https://github.com/rust-lang/rust/issues/61002
+#
+# Things work fine with -Cpanic=abort though.
+ifdef IS_MSVC
+WINDOWS_NOT_YET_SUPPORTED=true
+endif
```

Comment:
Why not just use `# ignore-msvc` below `# needs-profiler-support`?

Reply:
Cool! I didn't know that was an option.

I assume I could have also added it to the `mir-opt/instrument_coverage.rs` test instead of converting the check from an error to a warning (per my commit and comments earlier today).

I think I will still leave the options check as a warning (for MSVC) since I'll probably be debugging this a bit more in the mean time, and don't want to have to worry about re-enabling it temporarily while testing.

And the mir-opt test was working fine in MSVC prior to disabling the `-Zinstrument-coverage` flag.

But I'll use the new directive to disable the `run-make-fulldeps` test on MSVC, and remove that `ifdef`. Thanks.

Reply:
fixed